backend/scrapers/police_coders_groups/recompiler.py

```diff
@@ -14,11 +26,33 @@ def groupProcessing(filenames, boundaryfile):
     filename_csv = filenames["csv"]
     filename_demographics = filenames["demographics_legacy"]
 
-    varNm_geog_areaID = boundaryfile["ID_name"]
-    varNm_geog_areaNm = boundaryfile["area_name"]
+    varNm_geog_areaID =  boundaryfile['ID_name']
+    varNm_geog_areaNm = boundaryfile['area_name']
+    
+    polygon=getWelshBoundary()
+
+    features=getLSOABoundaries()
+
+    filteredList=filterBoundariesByWelsh(features, polygon)
+    LA_polygons=filteredList['filtered area polygons']
+    welshLAs=filteredList['filtered area features']
+
+    groups_QC=getQualityControlledList()
+
+    enterGroupsAsFeatures()
+
+    demographics=getDemographics()
+
+    output = countGroupsPerArea()
+
+    print("Message (groupProcessing): {} groups identified".format(pointsInLAs))
+
+    return (groups, output, geomWelshGrps, URLs)
 
-    # Import welsh boundaries
-    # load GeoJSON file containing sectors
+# Import welsh boundaries
+def getWelshBoundary():
```

Comment:
I am pretty sure I already saw a function with the **same** name elsewhere?

Reply:
i'll keep an eye out